bodo/tests/utils.py

```diff
@@ -229,15 +230,31 @@ def check_func(
     nullable float flag is on.
     - check_pandas_types: check if the output types match exactly, e.g. if Bodo returns a BodoDataFrame and python returns a DataFrame throw an error
     """
+    # If dataframe_library_enabled then run compiler tests as df library tests
+    # (replaces import pandas as pd with import bodo.pandas as pd)
+    # NOTE: This variable takes precedence over other variables
+    only_df_lib = bodo.dataframe_library_enabled
```

Comment:
This is overloading the meaning of this flag for testing which is ok for now but needs to be changed later when we turn this flag on by default with the release of the dataframe library.